src/NuGet.Clients/Options/GeneralOptionControl.cs

```diff
@@ -98,5 +101,26 @@ private void packageRestoreConsentCheckBox_CheckedChanged(object sender, EventAr
                 packageRestoreAutomaticCheckBox.Checked = false;
             }
         }
+
+        private void localsCommandButton_OnClick(object sender, EventArgs e)
+        {
+            var arguments = new List<string>();
+            arguments.Add("all");
+            var settings = ServiceLocator.GetInstance<ISettings>();
+            var localsCommandRunner = new LocalsCommandRunner(arguments , settings, clear:true, list:false);
+            try
+            {
+                localsCommandRunner.ExecuteCommand();
+            }
+            catch(ArgumentException ex)
+            {
+                MessageBox.Show(ex.Message, Resources.PopupTitle_ClearNuGetCache);
```

Comment:
Would user be able to understand these exception messages? is there any spec for this change? Also any design spec for dialog box?